src/index.html

```diff
@@ -11,6 +11,10 @@
       rel="stylesheet"
       href="./styles/main.scss"
     />
+    <script
+      src="scripts/main.js"
+      defer
+    ></script>
```

Comment:
The script tag is included twice in the HTML file. Consider removing this instance or the one at line 100 to avoid redundant loading of the same script.